src/methods.jl

```diff
@@ -48,10 +48,10 @@ mutable struct History
 end
 
 """
-    FDMethod
+    FiniteDifferencesethod
```

Comment:
Various instances of `FiniteDifferencesethod` should be `FiniteDifferenceMethod`

Reply:
I th

```suggestion
    FiniteDifferenceMethod
```